### ISSUE TITLE
split schedule builder link into two links

### ISSUE THREAD

**Issue by RuijieYu** [drewburns/student-admin-ext#16]
It might be helpful if the schedule builder link is separated into two links, one is the external website (possibly inside an iframe) and another that points to the enrollment page.

**Comment by drewburns** [comment]
Good idea on the iframe. Unfortunately Student Admin blocks all CORS so I couldn't load it inside the page, unless we figure out a work around.

On the two buttons: There already is a button for enroll. Which page were you referencing?
